src/index.js

```diff
@@ -1,17 +1,11 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import Calculator from './components/Calculator';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    {/* <App /> */}
+    <Calculator />
   </React.StrictMode>,
   document.getElementById('root'),
 );
```

Comment:
- [ ] <b>REQUIRED:</b> `App` component should be created in `src/component` directory as requested by the project. and it should be the one imported and used here, not `Calculator`.  `Calculator` should be imported and used in `App` component instead. Kindly fix it.